fern/overview/intro.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "Fern"
```

Comment:
```suggestion
title: "An overview of the Fern toolkit"
```

Reply:
Not sure if "Fern" was meant to be a placeholder, but reminder that currently, the title in docs.yml (`Fern | Documentation`) will be appended globally to titles set via front matter. So if the title is set to "Fern" in front matter, it will look like this when the page is rendered: `<title>Fern - Fern | Documentation</title>

Tip: In SEO, 60 chars is recommended max for a page title, so subtracting the char count of the docs.yml title plus spaces and dash (` - Fern | Documentation` in this case, which is 22 chars) gives the ideal max for the front matter title.